app/gui2/src/components/SvgIcon.vue

```diff
@@ -6,20 +6,20 @@
  */
 import icons from '@/assets/icons.svg'
 
-const props = defineProps<{ name: string }>()
+const props = defineProps<{ name: string; width?: number; height?: number }>()
 </script>
 
 <template>
-  <svg>
+  <svg :style="{ width: `${width}px`, height: `${height}px` }">
     <use :href="`${icons}#${props.name}`"></use>
   </svg>
 </template>
 
 <style scoped>
 svg {
-  width: 16px;
-  min-width: 16px;
-  height: 16px;
-  min-height: 16px;
+  width: var(--width, 16px);
```

Comment:
Width and height properties here are overwritten with dynamic `style` on the `<svg>`, am I right?

Reply:
oh... yeah, my bad. on second thought we can just use attribute fallthrough and avoid these props completely

Reply:
on second thought i should probably change the dynamic style to `--width` and `--height`, since we want both `width` and `min-width` to be the same